<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>710</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>varnam-tools</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-text-editor"/>
  </property>
  <property name="styleSheet">
   <string notr="true">/* General styles */
QWidget {
  background-color: #f5f5f5; /* Light gray background */
  font-family: Arial, sans-serif; /* Modern font */
}

QLabel {
  color: #333; /* Darker text for labels */
}

/* Text boxes with blue borders and rounded corners */
QLineEdit, QTextEdit {
  border: 1px solid #3498db; /* Blue border for text boxes */
  border-radius: 5px; /* Rounded corners for text boxes */
  padding: 5px;
  background-color: white;
}

/* Combo box style */
QComboBox {
  border: 1px solid blue; /* Light gray border for combobox */
  border-radius: 5px;
  padding: 5px;
  background-color: white;
}

QCheckBox {
  color: #333; /* Darker text for checkbox */
}

/* Specific styles for components */
QLabel#debugLabel {
  font-weight: bold; /* Bold text for debug label */
}



/* Specific styles for QTextEdit */
QTextEdit {
  background-color: #ffffff; /* White background */
  border: 1px solid #cccccc; /* Light gray border */
  padding: 5px; /* Padding */
}

/* Specific styles for QSpinBox */
QSpinBox {
  padding: 5px; /* Padding */
  border: 1px solid #cccccc; /* Light gray border */
  border-radius: 3px; /* Rounded corners */
  background-color: white; /* White background */
}

/* Specific styles for QFileDialog */
QFileDialog {
  background-color: #ffffff; /* White background */
  border: 1px solid #cccccc; /* Light gray border */
}

/* Specific styles for QMessageBox */
QMessageBox {
  background-color: #ffffff; /* White background */
  border: 1px solid #cccccc; /* Light gray border */
}

/* Specific styles for CustomHtmlDialog */
CustomHtmlDialog {
  background-color: #ffffff; /* White background */
  border: 1px solid #cccccc; /* Light gray border */
}

/* Specific styles for QMenuBar */
QMenuBar {
  background-color: #ffffff; /* White background */
  border: 1px solid #cccccc; /* Light gray border */
}

/* Specific styles for QMenu */
QMenu {
  background-color: #ffffff; /* White background */
  border: 1px solid #cccccc; /* Light gray border */
}

/* Additional component styles can be added as needed */
</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: light gray;
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select Scheme:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QComboBox" name="vstSelector">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="info">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="dialog-information"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="inputtextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial','sans-serif'; font-size:10.5pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arya'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>tokenizer suggestion limit -&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QSpinBox" name="dictSuggestionsSpinBox"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>dictionary suggestion limit -&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="tokenizerSuggestionsSpinBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>pattern dictionary suggestion limit -&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="patternDictSuggestionsSpinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="indicNumberCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>indic numbers</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tokenizerSuggestionsAlwaysCheckBox">
        <property name="text">
         <string>tokenizer suggestion always</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ignoreDuplicateTokenCheckBox">
        <property name="text">
         <string>ignore duplicate token</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="translit_buttons">
      <item>
       <widget class="QPushButton" name="transliterateButton">
        <property name="text">
         <string>Transliterate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reverseTransliterateButton">
        <property name="text">
         <string>Reverse Transliterate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>26</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="patternLineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="wordLabel">
        <property name="text">
         <string>Word:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="wordLineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="patternLabel">
        <property name="text">
         <string>Pattern:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>34</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="itrainingwordinput"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="trainingbuttons">
      <item>
       <widget class="QPushButton" name="trainButton">
        <property name="text">
         <string>Train with pattern=&gt;word</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="unlearnButton">
        <property name="text">
         <string>Unlearn(word)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="learnButton">
        <property name="text">
         <string>Learn(word)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="filebuttons">
      <item>
       <widget class="QPushButton" name="learnFromFileButton">
        <property name="text">
         <string>Learn From File(words only)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="trainFromFileButton">
        <property name="text">
         <string>Train From File(pattern word)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="text">
         <string>Export to vlf learning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importButton">
        <property name="text">
         <string>Import vlf learnings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>14</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="debugLabel">
      <property name="text">
       <string>Result</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="resultTextEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="version">
        <property name="text">
         <string>varnam_version</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/varnamproject&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0082fa;&quot;&gt;https://github.com/varnamproject&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
